.github/dependabot.yml

```diff
@@ -9,6 +9,7 @@ updates:
       # TODO(xacrimon): Update Firestore and solve deprecations.
       - dependency-name: cloud.google.com/go/firestore
       # Breaks backwards compatibility
+      - dependency-name: go.mongodb.org/mongo-driver
```

Comment:
FYI @jakule @greedy52, could one of you take a look at updating mongo-driver when you have the time?  (Assuming I got the right people this time.)

Reply:
there is some refactoring in the upstream driver. will do a PR to update it but likely it will take some time.

Reply:
Thanks. Please remove it from the ignore list when you do.

Reply:
I've made a [change](https://github.com/gravitational/teleport/pull/18646/files) to update to the new driver. Unfortunately, [this release v1.11.0](https://github.com/mongodb/mongo-go-driver/releases/tag/v1.11.0) deprecates Mongo versions older than v3.6, making all these tests fail:
https://github.com/gravitational/teleport/blob/7b11fbf3b8213a9fb4a963b58ce95e7632499832/lib/srv/db/access_test.go#L776-L779
with error
```
connection() error occurred during connection handshake: command failure: {"$err": "error connecting to the database\n\tserver at localhost:54395 reports wire version 5, but this version of the Go driver requires at least 6 (MongoDB 3.6)"}
```
I am leaning towards keeping the old driver. Not sure how many people are still on old Mongo though. @smallinsky what do you think?

Reply:
Definitely we should not drop the support for old versions -  probably some of our customer is using it. Recently we got issue related to mongoDB 3.6 https://github.com/gravitational/teleport/issues/18549 that confirm that old mongoDB versions are still in use.

Reply:
Raised https://github.com/gravitational/teleport/issues/18777 to track this. I think it's wise to plan a way forward, so we are not caught by surprise by this when a driver update becomes necessary.

@greedy52, once we know what the plan is, please update the TODO in dependabot.yml accordingly.